cmd/swarm/swarm-smoke/main.go

```diff
@@ -122,6 +124,18 @@ func main() {
 			Usage:       "whether to fetch content from a single node or from all nodes",
 			Destination: &single,
 		},
+		cli.IntFlag{
```

Comment:
you can infer this from `cluster-from` and `cluster-to` - let's use one of the other, everywhere.